variables.tf

```diff
@@ -0,0 +1,76 @@
+variable "name" {
+  default = "efs-backup"
+}
+
+variable "region" {
+  default = "us-east-1"
+}
+
+variable "vpc_id" {
+  default = ""
+}
+
+variable "datapipeline_config" {
+  type = "map"
+
+  default = {
+    instance_type = "t2.micro"
+    email         = ""
+    timezone      = "GMT"
+    period        = "24 hours"
+  }
+}
+
+variable "efs_ids" {
+  type = "map"
+
+  default = {}
+}
+
+variable "ssh_key_name" {
+  default = ""
+}
+
+variable "s3_bucket_expiration_days" {
+  default = "3"
+}
+
+variable "namespace" {
+  default = ""
+}
+
+variable "stage" {
+  default = ""
+}
+
+data "aws_availability_zones" "az" {}
```

Comment:
should be called `default`. This is not a variable so it should be moved to `main.tf`